### ISSUE TITLE
data clean

### ISSUE THREAD

**Issue by Teanoodle** [Teanoodle/Loan_Defaults_Classify#2]
import dataset

do the clean
( encoding the variables columns, find the correlation and pick variables)
 focus on the outliers

divide into training set and testing set (validation set if need)
